gateleen-routing/src/main/java/org/swisspush/gateleen/routing/Forwarder.java

```diff
@@ -385,6 +386,7 @@ private HttpClientRequest prepareRequest(final HttpServerRequest req, final Stri
             };
 
             cRes.exceptionHandler(exception -> {
+                LOG.warn("Failed to read upstream response for '{} {}'", req.method(), targetUri, exception);
```

Comment:
`exception` parameter is newer used when only having two `{}` placeholders. Or do I miss something?